[New] Adding new assets with updated aspect ratio

hazalarpalikli opened the pull request:
Today's museum of failures entry: Hazal accidentally [closed this ticket ](https://github.com/alphagov/govuk-brand-guidelines/pull/173)and we couldn't reopen it. We've opened a new pull request to get the work back on track.

## What 

This pull request converts all assets to 3:2 ratio and removes some remaining section highlights.

It also covers resizing some .svgs which have been added at a larger size. We're now uploading .svgs at 480px x 320px resolution to improve the experience for users who do not have .css enabled.

## Why

To make assets consistent across the guidelines

selfthinker commented:
Two more things I found which I hadn't added to the old #173...

## Uneven thirds on mobile

The cases of triplet wordmark / crown / other icon on some Logo system pages look odd when viewed on any mobile view with a narrower viewport than 390px (at least in my testing environment with Chrome on macOS).
The following screenshots are from the Samsung Galaxy S8+ emulation, which uses a viewport with a 360px width.

<img width="307" height="209" alt="Screenshot from the app page" src="https://github.com/user-attachments/assets/0d5c1bcb-1020-417f-988e-a2d14e4f9e1b" />
<img width="310" height="268" alt="Screenshot from the social page" src="https://github.com/user-attachments/assets/f5f8e475-6401-49c2-8e4b-75bb772fda3c" />

The first image is always bigger than the others because the word "Wordmark" doesn't fit into a third of the width.

Not sure how important that is. I'd say it's not a blocker.

Possible solution could be to maybe add a zero-width space (or is `<wbr />` still a thing?) between "Word" and "mark" so that it breaks at that point.
Another solution could be to get out of the three thirds layout into a single column or maybe two halves? The images get quite small at that point anyway.

## Missing "thematic break"

On the Brand hierarchy page the top four images had a grey background previously. They have now been made to look like the rest of the images on the page. That makes a confusing layout as it now looks like the bottom is a continuation from the top and that that is broken. It's missing some "thematic break" (not sure how to describe that). Also, the two types of images do different things and that is now lost.

<img width="273" height="461" alt="Screenshot of Brand hierarchy logos" src="https://github.com/user-attachments/assets/a562a2a6-6f71-4274-92ac-8850ceabd6a2" />

It could either get a visual difference between the two types (like with the grey background before) or maybe another heading to separate them?

selfthinker commented on src/typography/social/helvetica-neue-arial.svg:
(Optional) Although these had the same colours from the start, so is not something you have introduced, I only just realised that the colours don't have enough contrast. It's not super important, as this image is decorative and the names of the fonts are repeated in the body copy, but it would be nice to have more legible colours.

selfthinker reviewed (changes requested):
I've copied all of my comments over that are still open and were on #173.

CharlotteDowns commented on src/typography/font-specimen-bold.svg:
Thank you for finding it

selfthinker commented on src/typography/social/index.md:
```diff
@@ -28,62 +23,59 @@ Whilst there are many ways to build visual hierarchy, mixing weight and scale ac
 
 ### Headline styles
 
 Bold, over a maximum 5 lines
```
This was fixed in the latest changes.

selfthinker commented on src/typography/app/gds-transport.svg:
This has been fixed now.

selfthinker commented:
Thanks for all the fixes. I will approve this in a few minutes.
It would be good to get feedback for the things that have neither been resolved nor got feedback. I don't think those are a blocker, so don't need to be actioned for this to be merged, but it would be good to know what you're thinking about them.

Regarding the uneven thirds on mobile, I realised that this was not introduced by this PR, so doesn't need to be fixed by it. (When I wrote it I thought this was something new, but I checked what's currently in `main` and it's even worse in there.) I also found a second ocurrance. I will write a new GitHub issues for this. So, please ignore for the purpose of this PR.

The changes to the colours of the Arial and SF Pro images also don't need to be in here as they are something new.

The only bigger thing still open are some of the issues with whitespace in images. These are not a blocker, but it would be good to know if you plan to change that later, or if you are happy with how it looks.
Here are examples of what I mean:

<img width="708" height="689" alt="" src="https://github.com/user-attachments/assets/59726ad4-502e-4309-9258-827375b67fde" />
<img width="702" height="537" alt="" src="https://github.com/user-attachments/assets/efd3dca8-2814-4baa-b127-1ff826e61bcb" />
<img width="705" height="669" alt="" src="https://github.com/user-attachments/assets/8a8eac15-10e0-4588-9974-6e7067270af1" />
<img width="648" height="643" alt="" src="https://github.com/user-attachments/assets/7276fee7-6ef3-4a97-9eeb-2e218cf3fb54" />

One other new thing I found, which is not a blocker either: Was it intentional to leave the background of the font names white when they are on a light blue section highlight?

<img width="697" height="235" alt="" src="https://github.com/user-attachments/assets/c0afdcb1-a7d8-409e-84d3-2cde46690bc7" /> <img width="695" height="336" alt="" src="https://github.com/user-attachments/assets/e5a9593c-fae7-4c14-bc75-90e68a7c86d9" />

selfthinker reviewed (approved):
This is looking good. 👍 
The only thing left to do is to clean up the commit history.

CharlotteDowns commented:
Some of the whitespace @selfthinker flagged was intentional but we will re review and open a new pull request to correct if we feel it is needed.

owenatgov reviewed (approved):
Code wise I'm happy with this, pending the post-PR cleanup.

I've left a couple of comments that aren't critical. There are a few instances of assets without alt text but I'm notsure if this was intentional and has been addressed already elsewhere?

selfthinker commented on src/colour/app/index.md:
```diff
@@ -75,7 +75,9 @@ Indicative examples for illustrative purposes only.
-![Screenshot of the GOV.UK app, showing the start page of topics as greyed out.](./app-header1.png) ![Screenshot of the GOV.UK app, showing the Benefits topic page.](./app-header2.png) ![Screenshot of the GOV.UK app, showing the 'Pages you've visited' page.](./app-header3.png) ![Screenshot of the GOV.UK app, showing the Settings page.](./app-header4.png)
+![Screenshots of the GOV.UK app with the main content greyed out, showing the start page, Benefits, Pages you've visited and Settings pages.](./app-headers-1.png)
+
+![](./app-headers-2.png)
```
The first alt text describes the series of all four images (2x an image that each has 2 images in them = 4). Therefore the second doesn't need to describe them again. It's a technique that Calvin has used elsewhere within the brand guidelines website as well based on my recommendation.
Although giving each image different alt text would be valid, it would also be quite repetitive as most words would be the same.

CharlotteDowns committed:
Changes to assets

These changes include:
- converting most assets to 3:2 / 2:3 ratio (unless for specific reason needed adjustment)
- changing some assets to svg instead of png
- removing of some assets
- improving resolution of assets

CharlotteDowns committed:
Changes to layout

These changes include:
- removing section highlight from most places
- adding gridCell to some places
- adjusting text alignment to some places
- adding border to white videos

CharlotteDowns committed:
Changes to alt text and filetype

These changes include:
- calling svgs instead of pngs
- changing calls to assets that are now combined
- changing alt text to refer to combined assets

CharlotteDowns committed:
Changes to content

CharlotteDowns committed:
add video with starting thumbnail
